src/main/java/com/epam/university/java/core/task008/Task008Impl.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.university.java.core.task008;
+
+import java.util.Stack;
+
+/**
+ * Strings and braces.
+ */
+public class Task008Impl implements Task008 {
+    @Override
+    public boolean isValid(String sourceString) {
+        if (null == sourceString) return false;
```

Comment:
New lines and braces! In any case!

Reply:
i don't see the point,
what does it mean?..

Reply:
do you mean, that verification for null is excessive?

Reply:
I said, that you should always add curly braces:

```
if (null == sourceString) {
    return false;
}
```

Reply:
Got it